src/org/labkey/test/pages/study/specimen/ManageNotificationsPage.java

```diff
@@ -0,0 +1,175 @@
+package org.labkey.test.pages.study.specimen;
+
+import org.jetbrains.annotations.Nullable;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.RadioButton;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+
+public class ManageNotificationsPage extends LabKeyPage<ManageNotificationsPage.ElementCache>
+{
+    public ManageNotificationsPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ManageNotificationsPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        return beginAt(webDriverWrapper, webDriverWrapper.getCurrentContainerPath());
+    }
+
+    public static ManageNotificationsPage beginAt(WebDriverWrapper webDriverWrapper, String containerPath)
+    {
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("study-samples", containerPath, "manageNotifications"));
+        return new ManageNotificationsPage(webDriverWrapper.getDriver());
+    }
+
+    public ManageNotificationsPage setReplyTo(@Nullable String email)
+    {
+        if (email == null)
+        {
+            elementCache().replyToAdminRadio.check();
```

Comment:
Should this be uncheck?

Reply:
There are two radio buttons. This one says to use a use a "reply-to" address inherited from global site setting. The other one makes an input appear where you can specify an arbitrary email address. I decided to give `null` a special meaning instead of making 2 or 3 methods for setting the reply-to address.